srv/components/provisioner/scripts/install.sh

```diff
@@ -28,6 +28,7 @@ repo_url="file:///mnt/cortx"
 nodejs_tar=
 use_local_repo=false
 iso_downloaded_path="/opt/isos"
+LOCAL_IP=`ip -o addr | grep -v -w lo |awk '{print $4}' |head -1 |cut -c -14`
```

Comment:
I see LOCAL_IP is not getting used anywhere in the script, what's the purpose?

Reply:
to assign the local ip for nginx webserver for hosted rpm method